### PR TITLE
Fix alacritty colors

### PR DESCRIPTION
**Pull request by maheshsundaram**
Hi again! I don't know if this was intentional, but I found that the colors for alacritty were labeled differently than I expected. See the diff below for the inline notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,21 +41,21 @@ colors:
     black:   '#000000'
     red:     '#e33400'
     green:   '#5ccc96'
-    yellow:  '#b3a1e6'
+    yellow:  '#f2ce00'
```
Comment (maheshsundaram) [inline README.md:44]: This was actually the spaceduck 'purple'

File: README.md [hunk 2]
```diff
@@ -41,21 +41,21 @@ colors:
...
     blue:    '#00a3cc'
-    magenta: '#f2ce00'
-    cyan:    '#7a5ccc'
-    white:   '#686f9a'
+    magenta: '#ce6f8f'
```
Comment (maheshsundaram) [inline README.md:46]: This was actually the spaceduck 'yellow'

File: README.md [hunk 3]
```diff
@@ -41,21 +41,21 @@ colors:
...
+    cyan:    '#00a3cc'
```
Comment (maheshsundaram) [inline README.md:47]: This was actually the spaceduck 'purple2'

File: README.md [hunk 4]
```diff
@@ -41,21 +41,21 @@ colors:
...
+    white:   '#ffffff'
```
Comment (maheshsundaram) [inline README.md:48]: This was actually the spaceduck 'dark purple2'

File: README.md [hunk 5]
```diff
@@ -41,21 +41,21 @@ colors:
...
 
   # Bright colors
   bright:
     black:   '#686f9a'
     red:     '#e33400'
     green:   '#5ccc96'
-    yellow:  '#b3a1e6'
+    yellow:  '#f2ce00'
```
Comment (maheshsundaram) [inline README.md:55]: This was actually the spaceduck 'purple'

File: README.md [hunk 6]
```diff
@@ -41,21 +41,21 @@ colors:
...
     blue:    '#00a3cc'
-    magenta: '#f2ce00'
-    cyan:    '#7a5ccc'
+    magenta: '#ce6f8f'
```
Comment (maheshsundaram) [inline README.md:57]: This was actually the spaceduck 'yellow'

File: README.md [hunk 7]
```diff
@@ -41,21 +41,21 @@ colors:
...
+    cyan:    '#00a3cc'
```
Comment (maheshsundaram) [inline README.md:58]: This was actually the spaceduck 'purple2'

**Comment by pineapplegiant** [comment]
Hey! I appreciate the feedback a lot.

I agree, it is very confusing, and it is/was intentional. I don't know a better way? I wanted the colors to be more purple. I did the same thing for iTerm2.

Here are the two screenshots. First being the original that I have and the second with your changes.

original

changes

**Comment by pineapplegiant** [comment]
Colors are so subjective compared to code! Which do you prefer @maheshsundaram? I'm open to feedback.

**Comment by maheshsundaram** [comment]
@pineapplegiant Aha I see, thank you for explaining! You're right, I like it more purple as well!

**Comment by YoussefBouzekri** [comment]
btw is that an ls alternative you are using ? looks pretty colorful !

**Comment by pineapplegiant** [comment]
I think I’m using `exa` as my ls replacement @YoussefBouzekri 

https://the.exa.website/

Which is giving me the colors. There are others but it’s nice. I just alias ls to exa.

**Pull request closed without merging**